scriptsConverter.js

```diff
@@ -0,0 +1,44 @@
+document.getElementById("pageConverter").style.display = 'none';
+const result = document.getElementById("result");
+const exchangeRate = document.getElementById("exchangeRate");
+
+function showPageConverter() {
+    const element = document.getElementById("pageConverter");
+    element.style.display = "block";
+    
+}
+
+function getFromCurrency() {
+   return document.getElementById("selectFromValue");
+}
+
+function getToCurrency() {
+   return document.getElementById("selectToValue");
+}
+
+function getAmountMoney() {
+   return document.getElementById("inputAmountMoney");
+}
+
+function convertCurrency() {
+   let fromCurrency = getFromCurrency().value;
+   let toCurrency = getToCurrency().value;
+   let amount = getAmountMoney().value;
+   var requestURL = `https://api.exchangerate.host/latest?base=${fromCurrency}`;
+   var request = new XMLHttpRequest();
+   request.open('GET', requestURL);
+   request.responseType = 'json';
+   request.send();
+   request.onload = function test() {
+      var response = request.response.rates[toCurrency];
```

Comment:
Dałbym inną nazwę zmiennej, `request.response` to response, a Ty pobierasz wartość dla konkretnej waluty, więc może selectedCurrencyRate